recipes/cajun-jsonapi/all/test_package/main.cpp

```diff
@@ -0,0 +1,43 @@
+#include <cajun/json/reader.h>
+#include <cajun/json/writer.h>
+#include <cajun/json/elements.h>
```

Comment:
This does not match the official https://github.com/cajun-jsonapi/cajun-jsonapi/blob/da588ebde7d99a7d9b4c27e3411fdd4a7735837a/test.cpp#L30

```suggestion
#include <json/reader.h>
#include <json/writer.h>
#include <json/elements.h>
```

Reply:
this would break resiprocate conan build (#5310), see https://github.com/resiprocate/resiprocate/blob/6ccd5c361da7fbc508cced95334b9372672b7a17/repro/AccountingCollector.cxx#L8, I would then need to patch it

Reply:
Look at both repos it's the same commiter

The 2.0.3 release is one commit before https://github.com/cajun-jsonapi/cajun-jsonapi/commit/35c382d94223fc901861b54b04ce888ae42551d1

So he ski

https://github.com/cajun-jsonapi/cajun-jsonapi/blob/master/ReleaseNotes.txt

Reply:
Chrome not letting me edit.

I'd suggest keeping the packaging and add the include dirs... this way we can make both work

Does that work for you?

Reply:
> this would break resiprocate conan build (#5310), see resiprocate/resiprocate@6ccd5c3/repro/AccountingCollector.cxx#L8, I would then need to patch it

No, it won't. Both include and include/cajun will be used as include directory. Please, accept Prince suggestion, follow the upstream behavior.

Reply:
> No, it won't. Both include and include/cajun will be used as include directory.

@uilianries, you said that both `include` and `include/cajun` will be used as include directly, yet when I install this Conan dependency, I see the following folder structure:

```
// In ~/.conan/data/cajun-jsonapi/2.0.3/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9
- /include
  - /json 
``` 

With this folder structure (no `cajun` folder) I'm unable to compile [`resiprocate`](https://github.com/resiprocate/resiprocate/blob/6ccd5c361da7fbc508cced95334b9372672b7a17/repro/AccountingCollector.cxx#L8) as Tomas correctly pointed out. 

Can you please explain why you insisted on removing `cajun` subfolder and what exactly do you mean by "follow the upstream behavior"? Upstream of what?

As far as I understand, given a packaged dependency, it is possible to relatively easy shorten include path (use `#include "json/..."` instead of `#include "cajun/json/..."`) but I'm not sure the reverse is possible. Am I missing something here?

P.S. I also wanted to point out that Cajun has been shipped as [RPM package](https://centos.pkgs.org/7/epel-x86_64/cajun-jsonapi-devel-2.0.3-2.el7.noarch.rpm.html) with `cajun` folder on the include path, so current Conan recipe prevents smooth transition from RPM-based dependency management to Conan-based dependency management.